### DISCUSSION TITLE
Changes, such as product translations, are not saved

### DISCUSSION THREAD

**Discussion by Polen** [PrestaShop/PrestaShop#32874] [Q&A]
After installing PS 8.0.4 I encounter the following problem.
Changes are not saved in several places.
The creation of a product goes well, but translations are not saved afterwards.

I have all write permissions set to 755, on advice from another forum.
What I see next is the warning:
Changed/missing files have been detected.
updated files (1)
LICENSES

I have refreshed this file again, but it gives the same warning.

If anyone knows a solution, a statue in my backyard is one of the possibilities.

**Comment by Polen** [comment]
Also Theme & Logo > Theme images are not updated.
Anyboduy any clue?

**Comment by matks** [comment]
I am sorry but this without being able to look at your shop and analyze it I don't think anybody can help you.

I can suggest
- to look at error logs
- to check this page https://www.prestashop-project.org/support/

**Comment by kpodemski** [comment]
Hi @Polen 

What is your PHP version? There may be a problem there. It could also be some extra cache mechanism on your server or your PrestaShop server, etc. It would be best to hire somebody to look at the problem directly in your store.
